Pear/Controller/FeedbackViewController.swift

```diff
@@ -219,16 +219,9 @@ class FeedbackViewController: UIViewController {
         animateView(duration: 0.4, delay: 1.6, previousY: askDetailLabel.center.y, yOffset: 60, animateView: detailTextView)
         animateView(duration: 0.6, delay: 2.0, previousY: finishButtonY, yOffset: 0, animateView: finishButton)
     }
-
-    @objc private func closeFeedback() {
-        self.dismiss(animated: true, completion: nil)
-    }
-
+    
     @objc private func finishFeedback() {
-        print("finish feedback")
-        responses.append(detailTextView.text)
         print(responses)
-        self.dismiss(animated: true, completion: nil)
     }
```

Comment:
Did you mean to have this function only print the responses?

Reply:
since the feedback routes are not set up on the backend yet, this view controller is not used. I am just using printing for now as a placeholder.